apps/site-landing/src/containers/Subscribe/styles.module.less

```diff
@@ -1,46 +1,40 @@
 @import "~@thecointech/site-semantic-theme/variables.less";
 
-#subscribeBlock{
-    display:flex;
-    flex-direction: column;
-    justify-items: center;
-    color: white;
-    button{
-      margin-left: 20px;
-    }
-  }
-
-  .subscribe {
-    position: absolute;
-    left: 2%;
-  }
+#subscribeBlock {
+  padding-top   : 80px;
+  display       : flex;
+  flex-direction: column;
+  justify-items : center;
+  color         : white;
 
-  .subContainer {
-    margin: auto;
+  .message {
+    margin    : auto;
```

Comment:
Not a big fan of those. It was a while back but some browser refused to interpret those.

Reply:
neither.  Also, the auto formatter does a really bad job of LESS mixin formatting.  So will probably ditch it as soon as I have time to look for alternatives

Reply:
Did you take a look at the PR I opened? https://github.com/thecointech/thecoin/pull/344

Reply:
yes, that's where I plan to start :-).  I figured I'll get to a decent checkpoint, then re-format all the files in the repo so we don't get so much noise in PR's